rust/scx_utils/src/topology.rs

```diff
@@ -238,10 +238,12 @@ impl Topology {
         }
 
         let nr_cpus_online = span.weight();
+        let cpu_online_ids = cpu_online_ids(&span);
         Ok(Topology {
             nodes,
             span,
             nr_cpus_online,
+            cpu_online_ids,
```

Comment:
Can't it just be Cpumask?

Reply:
Also, can you name it `cpus_online`?

Reply:
Hi @htejun,

the idea was to have a precomputed value that could be used instead of iterating over `nr_cpu_ids` and then calling `test_cpu()`. 

However, the Cpumask is already sufficient for this. 

I'll close this draft.